.ci/release.sh

```diff
@@ -71,9 +71,9 @@ function release::git_setup() {
 
 function release::publish_charts() {
     release::git_setup
-    git clone https://${GITHUB_TOKEN}@github.com/apache/pulsar
-    cd pulsar
-    git checkout asf-site
+    git clone https://${GITHUB_TOKEN}@github.com/apache/pulsar-site
```

Comment:
the default branch is `asf-site` for now, However, the JIRA has been submitted for modification, it's recommend change it to `git clone -b main https://${GITHUB_TOKEN}@github.com/apache/pulsar-site` @tuteng

Reply:
Of course, it doesn't matter, because you switched branches to `asf-site-next`